### ISSUE TITLE
Nasty bug with Aliases of named imports

### ISSUE THREAD

**Issue by asfktz** [Galooshi/import-js#407]
Hi, What's up? (:
Found some nasty bug

Steps to reproduce:

1. Create 2 files:

```js
// numbers.js
export const one = 1;
export const two = 2;
````

```js
// index.js
import { one as test, two } from './numbers';

console.log(test);
console.log(two);
````

2. Remove `console.log(two);`
```js
// index.js
import { one as test, two } from './numbers';

console.log(test);
````
3. Run: fix all imports

Results:
```js
// index.js
import { test } from './numbers';

console.log(test);
````

Notice how `{ one as test }` is now `{ test }`

It's important to note that it only happens when you remove another named module from the same import statement, but not if it's from a different one.

For example:

```js
// index.js
import { one as test } from './numbers';
import letters from './letters';


console.log(test);
console.log(letters);
```

Remove `console.log(letters);`

```js
// index.js
import { one as test } from './numbers';
import letters from './letters';

console.log(test);
```

Hit FIX.

Results:

```js
// index.js
import { one as test } from './numbers';

console.log(test);
```

This is current.

----

Real world example:

![333](https://cloud.githubusercontent.com/assets/199747/26729960/1170d4ca-47b8-11e7-90c5-9697598417e0.gif)

**Comment by trotzig** [comment]
Ah, thanks for the report and for the excellent repro instructions! I have a fix coming up.

**Comment by asfktz** [comment]
Thanks @trotzig (:

**Comment by trotzig** [comment]
The fix was released in v2.3.1, and in atom-import-js in [0.7.1](https://github.com/Galooshi/atom-import-js/releases/tag/v0.7.1)

**Comment by asfktz** [comment]
Tested it on Sublime, works great!  🎉
